### PR TITLE
Fix for flaky test

### PR DESCRIPTION
**Pull request by vytis**
It seems the test introduced in #186 is quite flaky, trying to make it more stable

### CODE REVIEW

File: Source/Model/Conversation/ZMConversationSecurityLevel.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+extension ZMConversationSecurityLevel: CustomDebugStringConvertible {
```
Comment (mikeger) [inline Source/Model/Conversation/ZMConversationSecurityLevel.swift]: Additional debug utility? Does not seem to be used.

Comment (vytis) [inline Source/Model/Conversation/ZMConversationSecurityLevel.swift]: It's used when calling `.debugDescription`, which happens when `XCTAssert` fails. Without it we get this sort of failure:

```
XCTAssertEqual failed: ("ZMConversationSecurityLevel") is not equal to ("ZMConversationSecurityLevel") - 
```

Comment (mikeger) [inline Source/Model/Conversation/ZMConversationSecurityLevel.swift]: Ah nice

Comment (mikeger) [inline Source/Model/Conversation/ZMConversationSecurityLevel.swift]: The thing could also just display it's `rawValue` per default. I always thought it's the another annoying apple testing bug.

**Pull request merged**